package.json

```diff
@@ -354,7 +354,7 @@
     "@metamask/snaps-sdk": "^6.16.0",
     "@metamask/snaps-utils": "^8.9.1",
     "@metamask/solana-wallet-snap": "^1.2.0",
-    "@metamask/transaction-controller": "^43.0.0",
+    "@metamask/transaction-controller": "npm:@metamask-previews/transaction-controller@44.0.0-preview-ee77a6d9",
```

Comment:
Any reason why you're using the preview build instead of the release? 😄

Reply:
This is our standard practice so we can QA before merging the core PR and releasing the controller.

The adopting PR (this one) gets updated with the real version before merging.